Northstar.Client/mod.json

```diff
@@ -17,18 +17,22 @@
 			"Name": "filter_hide_protected",
 			"DefaultValue": "0"
 		},
-		{
-			"Name": "filter_map",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_gamemode",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_mods",
-			"DefaultValue": "0"
-		},
+        {
+            "Name": "filter_map",
```

Comment:
Unintentional indentation change? Like only default value was changed but more stuff shows up in diff

Reply:
Is this an issue that I need to fix, or is it just a remark ?
And no, I don't see why GitHub considers every line as replaced instead of modified. 
Also, my indentations are done with tabs and a size of 4.

Reply:
That's an issue that should be resolved. Forgot I could just use web version of vscode to check diff. Anyway looks likes it caused by replacing tabs with spaces:

![image](https://github.com/R2Northstar/NorthstarMods/assets/40122905/e9db121b-f95b-4f62-bab0-44308e91394f)

Reply:
Okay It's fixed